### ISSUE TITLE
Doesn't start

### ISSUE THREAD

**Issue by devin1229** [fontforge/fontforge#2538]
I just downloaded and installed the app to my Applications folder and nothing happens when I try to launch it. I have XQuartz 2.7.7 installed.

**Comment by wollmers** [comment]
Same here. OSX 10.9.5, XQuartz 2.7.7.

**Comment by ultrasquid** [comment]
XQuartz has been updated to 2.7.8, and among the changes is something
dealing with a launching issue on Yosemite. Try updating your XQuartz and
then report whether this problem changes or not.

On Tue, Oct 20, 2015 at 2:59 PM, Helmut Wollmersdorfer <
notifications@github.com> wrote:

> Same here. OSX 10.9.5, XQuartz 2.7.7.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fontforge/fontforge/issues/2538#issuecomment-149715213>
> .
>



-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by wollmers** [comment]
I still have XQuartz 2.7.8, but the August 2015 version of fontforge (the prepackaged .dmg) shows only a progress bar "searching fonts" which disappears. It's not on Yosemite (10.10), it's on Mavericks (10.9). Fontforge version 31-July-2012 is still working.

**Comment by devin1229** [comment]
Yup. Just updated Quartz to 2.7.8 and it still doesn't launch.

**Comment by ultrasquid** [comment]
FontForge should start after that progress bar disappears, but there can be
a delay of up to several minutes. How long have you waited after the
progress bar disappears before giving up?

This is known and I'm fairly certain that's already been raised as an
issue, but I'll check on it.

On Wed, Oct 21, 2015 at 1:13 PM, devin1229 <notifications@github.com> wrote:

> Yup. Just updated Quartz to 2.7.8 and it still doesn't launch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fontforge/fontforge/issues/2538#issuecomment-150012011>
> .
>



-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by Em2bee** [comment]
Hi, 

Same here... Doesn't launch
 I've got the latest version of FontForge (just downloaded it today) and XQuartz 2.7.8. and the latest version of OSX El Capitan (10.11.1)

It doesn't even give me a dialog window (except the one telling me «this app is loaded for the internet») Just bounce quickly in my dock and disappear after a sec.

**Comment by rmcb** [comment]
Ditto. It just will not open. I am on Yosemite, and have the latest Quartz.

**Comment by ronalson** [comment]
I have the same issue here with XQuartz 2.7.8 and OSX 10.11. Whenever I try to open the application the icon pops and close immediately.

**Comment by nemoDreamer** [comment]
:+1:

Updated XQuartz, running El Capitan:

Opening FontForge only gives the "downloaded from we, sure to open?" dialog, then spends a while "verifying FontForge", then nothing: icon disappears from dock, never see the "trusted developer" dialog.

Will try downloading a daily as recommended in #2555 .

**Comment by nemoDreamer** [comment]
Ah, saw this note in the Aug 2015 Mac release:

https://github.com/fontforge/fontforge/releases/tag/20150824

> Note that, for currently unknown reasons, the Macintosh build published here quits immediately after launching from the Finder. Launch it from the Terminal (`/Applications/FontForge.app/Contents/MacOS/FontForge`) instead.

**Comment by nemoDreamer** [comment]
Tried the "via Homebrew / Cask" method as well, but was still missing some Python libs similar to #2535, and also needed to run cask w/ the `--appdir=/Applications` flag, btw...

**BUT** then I downloaded the HEAD zip from the "Daily Dev" section of http://fontforge.github.io/en-US/downloads/mac/, and that opened up perfectly right from my /Applications folder (no Terminal needed). Yay!

**Comment by ultrasquid** [comment]
I have to question whether the HEAD zip from the "Daily Dev" section
of http://fontforge.github.io/en-US/downloads/mac/

is actually a daily build. Selecting Help>About shows the Lib Version date
as 23:52 PDT 26-May-2015. That's several months ago.
<http://fontforge.github.io/en-US/downloads/mac/>

On Mon, Nov 9, 2015 at 9:25 AM, Philip Blyth <notifications@github.com>
wrote:

> Tried the "via Homebrew / Cask" method as well, but was still missing some
> Python libs similar to #2535
> <https://github.com/fontforge/fontforge/issues/2535>, and also needed to
> run cask w/ the --appdir=/Applications flag, btw...
>
> *BUT* then I downloaded the HEAD zip from the "Daily Dev" section of
> http://fontforge.github.io/en-US/downloads/mac/, and that opened up
> perfectly right from my /Applications folder (no Terminal needed). Yay!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fontforge/fontforge/issues/2538#issuecomment-155131794>
> .
>



-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by wollmers** [comment]
Yes, been there, done this. The Daily is form May 2015 and is unstable, i.e. it crashes often. I stepped back to a 2012 version.

**Comment by nemoDreamer** [comment]
:disappointed:

**Comment by 8tracker** [comment]
Well… this is dumb.

I was having the same issue of verifying the FontForge app. I went to System Preferences>Security & Privacy and checked “Anywhere” under “Allow apps downloaded from:” This didn't fix it.

Then I used AppZapper to delete the FontForge app in hopes of replacing it with the 2012 version (which I couldn't find). I then pulled the deleted FontForge app out of the trash and back to my Applications folder and now it launches after the one-time setup dialog box.

I have no idea what any of this means.
